### DISCUSSION TITLE
Theme dont enabling

### DISCUSSION THREAD

**Discussion by zibertszn** [ohmyzsh/ohmyzsh#10630] [Help] [answered]
![image](https://user-images.githubusercontent.com/72556979/151325929-548a1776-52f1-43e3-8771-da94c67d303f.png)
i have this lines of code but theme dont enabling.
P.S other cfg files do not have "ZSH_THEME" line

**Comment by mcornella** [comment, marked as answer]
Do you `source $ZSH/oh-my-zsh.sh` in `$HOME/.config/zsh/.zshrc`? If not, Oh My Zsh is not actually loaded.
